packer/scylla_install_image

```diff
@@ -19,6 +19,7 @@ from subprocess import run, PIPE, STDOUT
 
 my_env = os.environ.copy()
 my_env['DEBIAN_FRONTEND']='noninteractive'
+apt_keys_dir = '/etc/apt/keyrings'
```

Comment:
Let's also add a parameter with APT_KEY value

Reply:
It's a one ref only so no really need for variable